### ISSUE TITLE
run error in IDE

### ISSUE THREAD

**Issue by chris-wjn** [SilentChaos512/Silent-Gear#431]
## Versions

- **Silent Gear**: 2.6.30
- **Silent's Gems**: N/A
- **Silent Lib**: 4.9.6
- **Forge**: 36.2.0
- **Modpack**: N/A
- **Optifine Installed**: No

## Expected Behavior
Intellij runClient to run properly.

(Description here)

## Actual Behavior
Throws an error and crashes.

(Description here)

## Links/Images
Running the runClient option in intelliJ causes the terminal to throw this error:

Mixin apply failed silentgear.mixins.json:MixinItemEntity -> net.minecraft.entity.item.ItemEntity: org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException @Shadow method func_92059_d in silentgear.mixins.json:MixinItemEntity was not located in the target class net.minecraft.entity.item.ItemEntity. Using refmap silentgear.refmap.json

At which point a few more lines of errors come up, then it crashes.

## Steps to Reproduce the Problem
implement silentgear and silentlib using the following code in build.gradle:

repositories {
maven {
        url "https://www.cursemaven.com"
    }
}
dependencies {
    implementation ("curse.maven:silentgear-297039:3386805") {
        exclude module: 'forge'
        exclude module: 'jei-1.16.3'
        exclude module: 'silent-lib-1.16.3'
        exclude module: 'curios-forge'
    }
    implementation ("curse.maven:silentlib-242998:3235738") {
        exclude module: "forge"
    }
}
This is done as it is in the iafgear addon.

**Comment by SilentChaos512** [comment]
Try adding the following lines to the build.gradle's run configs: https://github.com/SilentChaos512/Silent-Gear/blob/1.17.x/build.gradle#L72-L73. Might be the same issue I have with Patchouli.

**Comment by chris-wjn** [comment]
Added the lines, still getting same error. Maybe it's something to do with how I'm adding the dependencies? It's different than what's on your wiki, but the maven repos you have linked are not valid pages.

**Comment by SilentChaos512** [comment]
Hmm, pretty sure the information on the README.md is correct...

**Comment by chris-wjn** [comment]
But if i go to the urls that are provided in the README.md for the repositories to add, they lead to error 404.

**Comment by Cwalker60** [comment]
I ran into a similar issue getting silent-gear to run through those same dependencies on cursemaven for intelj.
However, I got mine to build by adding fg.deobf("...") {} after each implementation of silent-gear and silent-lib.

so implementation ("curse.maven:silentgear-297039:3386805") would change too
implementation fg.deobf("curse.maven:silentgear-297039:3386805")
same idea with the implementation for the silent-lib dependency too.
